tests/expose-values.js

```diff
@@ -0,0 +1,43 @@
+//Test things dealing with the initialValues prop
+import React from 'react';
+import { spy } from 'sinon';
+import Form from '../src/form';
+import { shallow } from 'enzyme';
+
+const Input = ({ error, ...props }) =>
+  error ? <p className="error">{error}</p> : <input {...props} />;
+
+test('Form passes values to onValues', () => {
+  const onValues = spy();
```

Comment:
ooo i used it back before i used jest and didnt bother to look and see. maybe ill make a pr where i remove it soon. You're welcome to if you want as well.